src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.html

```diff
@@ -68,6 +68,11 @@
                   </div>
                 </td>
               </tr>
+              <tr *ngIf="user.mfa_ids && user.mfa_ids.length">
+                <td i18n
+                    class="bold">MFAs(Id)</td>
+                <td>{{ user.mfa_ids }}</td>
```

Comment:
The human-readable version of arrays should be rather `"elem1, elem2, elem3"` rather than `"elem1,elem2,elem3"`.
```suggestion
                <td>{{ user.mfa_ids.join(', ') }}</td>
```
However this means adding a method to a template expression, which is highly discouraged for performance reasons. We have a custom `join` pipe implemented for this:
```suggestion
                <td>{{ user.mfa_ids | join }}</td>
```